docs/manifest.json

```
@@ -463,169 +729,206 @@
+                              },
+                              {
+                                "title": "Disable Account Portal",
+                                "href": "/docs/customization/account-portal/disable-account-portal"
+                              }
+                            ]
+                          ]
                         }
                       ]
                     ]
-                  }
-                ]
-              ]
-            },
-            {
-              "title": "Component Reference",
-              "collapse": true,
-              "icon": "box",
-              "items": [
-                [
-                  { "title": "Overview", "href": "/docs/components/overview" },
-                  {
-                    "title": "`<ClerkProvider>`",
-                    "wrap": false,
-                    "href": "/docs/components/clerk-provider"
                   },
                   {
-                    "title": "Authentication Components",
+                    "title": "Appearance prop",
+                    "collapse": true,
                     "items": [
                       [
                         {
-                          "title": "`<SignIn />`",
-                          "wrap": false,
-                          "href": "/docs/components/authentication/sign-in"
+                          "title": "Overview",
+                          "href": "/docs/customization/overview"
                         },
                         {
-                          "title": "`<SignUp />`",
-                          "wrap": false,
-                          "href": "/docs/components/authentication/sign-up"
+                          "title": "Layout",
+                          "href": "/docs/customization/layout"
                         },
                         {
-                          "title": "`<GoogleOneTap />`",
-                          "wrap": false,
-                          "href": "/docs/components/authentication/google-one-tap"
+                          "title": "Themes",
+                          "href": "/docs/customization/themes"
+                        },
+                        {
+                          "title": "Variables",
+                          "href": "/docs/customization/variables"
                         }
                       ]
                     ]
                   },
+                  { "title": "Localization", "href": "/docs/customization/localization" },
                   {
-                    "title": "User Components",
+                    "title": "Custom Pages",
+                    "collapse": true,
                     "items": [
                       [
                         {
-                          "title": "`<UserButton />`",
-                          "wrap": false,
-                          "href": "/docs/components/user/user-button"
+                          "title": "User Profile",
```

is there a reason we removed the component syntax?

```suggestion
                          "title": "`<UserProfile />`",
```